examples/js/cameras/CombinedCamera.js

```diff
@@ -1,31 +1,38 @@
 /**
  *	@author zz85 / http://twitter.com/blurspline / http://www.lab4games.net/zz85/blog
  *
- *	A general perpose camera, for setting FOV, Lens Focal Length,
+ *		A general perpose camera, for setting FOV, Lens Focal Length,
  *		and switching between perspective and orthographic views easily.
  *		Use this only if you do not wish to manage
  *		both a Orthographic and Perspective Camera
  *
  */
 
 
-THREE.CombinedCamera = function ( width, height, fov, near, far, orthoNear, orthoFar ) {
+THREE.CombinedCamera = function ( width, height, fov, near, far, hyperfocusOffset, hyperfocusScale ) {
```

Comment:
These should not be creation parameters.  Rather they should only be set in the constructor to 0 and 0.5 respectively.